src/quo2/components/dropdowns/network_dropdown/style.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.dropdowns.network-dropdown.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container-border-color
+  [{:keys [state blur? theme pressed?]}]
+  (let [default-color (if blur?
+                        colors/neutral-80-opa-5
+                        (colors/theme-colors colors/neutral-30 colors/neutral-70 theme))
+        active-color  (if blur?
+                        colors/neutral-80-opa-10
+                        (colors/theme-colors colors/neutral-40 colors/neutral-60 theme))]
+    (cond
```

Comment:
can be `(condp = state)`
And the clauses should just be
`:disabled default-color`
`:default (if pressed? active-color default-color)`

Reply:
@J-Son89 are we going to add blur color to theme-colors?